includes/admin/settings/class-settings-bootstrap.php

```diff
@@ -350,13 +350,42 @@ private static function settings_general() {
                 'group' => 'general/advanced',
             )
         );
+		if ( ! is_plugin_active( 'font-awesome/index.php' ) || ( ! class_exists( '\FortAwesome\FontAwesome_Loader' ) ) ) {
+			wpbdp_register_setting(
+				array(
+					'id'    => 'fontawesome-enabled',
+					'name'  => _x( 'FontAwesome styles?', 'settings', 'business-directory-plugin' ),
+					'desc'  => wp_kses_post(
+						sprintf(
+							/* translators: %s: addon name */
+							'<span>' . __( 'Install %1$s for FontAwesome styles?', 'business-directory-plugin' ) . '</span>',
+							'<a href="https://wordpress.org/plugins/font-awesome" target="_blank" rel="noopener nofollow">Font Awesome</a> ',
```

Comment:
For the field icons it might be better to leave it in the settings

Reply:
Can you tell me why you would like it in the settings?

Reply:
The fonts will not only be used in the social field type when the field icons are implemented. This will be a common setting to be used in the field icons display and would work better as a general setting

Reply:
The problem is, the spot it's in right now just doesn't make sense. If we include it in the settings, it should be at the bottom of the appearance tab.

Reply:
All changed to tabs

Reply:
Also settings moved